src/main/resources/reference.conf

```diff
@@ -74,6 +78,7 @@ resourceTypes =
           roleActions = ["launch_notebook_cluster"]
         }
       }
+      reuseIds = false
```

Comment:
for billing projects we should make sure to override this value to `true` for fiabs so we don't break gpalloc

Reply:
I would just set it to true here because we only allow delete in non-prod envs anyway